store/storetest/user_store.go

```diff
@@ -2764,7 +2764,7 @@ func testUserStoreSearchNotInChannel(t *testing.T, ss store.Store) {
 	ch1 := model.Channel{
 		TeamId:      tid,
 		DisplayName: "NameName",
-		Name:        "zz" + model.NewId() + "b",
+		Name:        NewTestId() + "b",
```

Comment:
Please remove the `b` here and everywhere else as well.

Reply:
I did.